### PR TITLE
Move fields from Journalpost to AbstraktRegistrering in web services v1

### PR DESCRIPTION
**Pull request by vulkodava**
* journalansvarlig
* journalansvarligBrukerIdent

### CODE REVIEW

File: version1/02 - Object types.md [hunk 1]
```diff
@@ -404,49 +404,51 @@ The following notation is used in the tables below:
...
+|           | string    | publiseringsstatus               |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | Allowed values: NOT_PUBLISHED, PUBLISHED_METADATA, PUBLISHED_METADATA_AND_DOCUMENTS                                                                         |
+| M300      | string    | dokumentmedium                   |    x     |      x      | dokumentmedium     |  |  x   |  x   |   x   |      |      |        |                                                                                                                                                             |
+|           | string    | journalansvarlig                 |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | <p>&bull; The name of the person responsible for this Journalpost (similar to M306 Saksmappe#saksansvarlig)</p>                                             |
```
Comment (ivaylomitrev) [inline version1/02 - Object types.md]: I was going to mention that the comment specifies Journalpost, but then I saw that the same applies to meeting record too.

Let's not edit that for now and fix it in another PR.

**Review by ivaylomitrev — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
You still have my approval after the latest changes.

File: version1/02 - Object types.md (hunk 1 above)
Comment (vulkodava) [reply version1/02 - Object types.md]: Must be addressed with https://github.com/documaster/noark5-web-services/pull/64/commits/62acd7c07915db4f427c3a087834c00e7271e868

**Review by ivaylomitrev — APPROVED** [review]

**Review by venelinkotsev — APPROVED** [review]

**Comment by dkumanov** [comment]
I think you will need to rebase on the latest develop and add the fields to `Arkivnotat` as well.

File: version1/02 - Object types.md [hunk 2]
```diff
@@ -404,49 +404,51 @@ The following notation is used in the tables below:
...
+|           | string    | publiseringsstatus               |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | Allowed values: NOT_PUBLISHED, PUBLISHED_METADATA, PUBLISHED_METADATA_AND_DOCUMENTS                                                                               |
+| M300      | string    | dokumentmedium                   |    x     |      x      | dokumentmedium     |  |  x   |  x   |   x   |      |      |        |                                                                                                                                                                   |
+|           | string    | journalansvarlig                 |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | <p>&bull; The name of the person responsible for this Basisregistrering (similar to M306 Saksmappe#saksansvarlig)</p>                                             |
```
Comment (dkumanov) [inline version1/02 - Object types.md]: Unlike `Journalpost` the fields can be null for the other types and they don't have a default value so the x-marks need to be removed.

**Review by dkumanov — CHANGES_REQUESTED** [review]

File: version1/02 - Object types.md (hunk 2 above)
Comment (vulkodava) [reply version1/02 - Object types.md]: Great catch :) Fixed with https://github.com/documaster/noark5-web-services/pull/64/commits/d39d319323ac0605789129aad5035ebccd4054a9 and https://github.com/documaster/noark5-web-services/pull/64/commits/d6a37471b33bca33fbceaa9210c086197a71569c

With the same commit I have added documentation for the same fields and for the Arkivnotat.

**Review by ivaylomitrev — APPROVED** [review]

**Review by dkumanov — APPROVED** [review]

**Pull request merged**